### PR TITLE
create first user model

### PR DESCRIPTION
**Pull request by jonnyalexbh**
## Summary
- User model

## Trello Card
https://trello.com/c/DEi5x3TI/26-modelo-de-usuario

### CODE REVIEW

File: app/models/user.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      firstName: {
```
Comment (jersonlopez) [inline app/models/user.js]: in coding the attributes are named in camelCase, but at db level they must be named in snake_case

File: app/models/user.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        allowNull: false,
+        type: DataTypes.STRING,
+        unique: true
+      },
+      password: {
+        type: DataTypes.STRING,
+        allowNull: false
+      }
+    },
+    {}
```
Comment (jersonlopez) [inline app/models/user.js]: did you read something about `underscored` `freezeTableName`... those attributes are part of the db configuration

File: migrations/migrations/20190724232643-create-user.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('users', {
```
Comment (jersonlopez) [inline migrations/migrations/20190724232643-create-user.js]: nowhere in the model says the table is called `users`, in the migrations is good, but in the model missing it

File: migrations/migrations/20190724232643-create-user.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      firstName: {
```
Comment (jersonlopez) [inline migrations/migrations/20190724232643-create-user.js]: in the db the attributes are named in snake_case

File: app/models/user.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      FirstName: {
```
Comment (jersonlopez) [inline app/models/user.js]: do you really know what's camelCase?

Comment (jonnyalexbh) [reply app/models/user.js]: It is UpperCamelCase or lowerCamelCase ?

**Pull request merged**